internal/cmd/flink/command_statement_list.go

```diff
@@ -33,12 +37,15 @@ func (c *command) statementList(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	statements, err := client.ListStatements(environmentId, c.Context.LastOrgId)
+	statements, err := client.ListAllStatements(environmentId, c.Context.LastOrgId)
 	if err != nil {
 		return err
 	}
 
+	sortStatementsByCreationTime(statements)
+
 	list := output.NewList(cmd)
+	list.Sort(false) // disable the default sort
```

Comment:
because that sorts it by some other logic: https://github.com/confluentinc/cli/blob/8f102997533171ba5ddfa31b3582067eb2daeeba/internal/pkg/output/table.go#L96

Reply:
and if I don't disable it here it will overwrite my sort order again

Reply:
iirc, it will only sort by creation time if the creation time is the first element (this might be a good thing to do)

Reply:
Just tried it and that works! It's now included in the output though, but I guess that is alright (might even be better to actually see it)